apps/ledger-live-desktop/src/newArch/WalletSync/Flows/Synchronize/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Flex } from "@ledgerhq/react-ui";
+import React from "react";
+import { useDispatch } from "react-redux";
+import { setFlow } from "~/renderer/actions/walletSync";
+import { Flow, Step } from "~/renderer/reducers/walletSync";
+import { useFlows } from "../useFlows";
+import SynchronizeModeStep from "./01-SynchModeStep";
+import SynchWithQRCodeStep from "./02-QRCodeStep";
+import PinCodeStep from "./03-PinCodeStep";
+
+const SynchronizeWallet = () => {
+  const dispatch = useDispatch();
+
+  const { currentStep, goToNextScene } = useFlows({
+    flow: Flow.Synchronize,
+  });
+
+  const goToActivation = () => {
+    dispatch(setFlow(Flow.Activation));
```

Comment:
I think that when we change the flow we should also be able to change the step too. Because a step belongs to a flow.
``` 
dispatch(setFlow(Flow.ManageBackups));
dispatch(setStep(Step.ManageBackupStep));
```
Maybe we could have something like this instead:
`dispatch(setFlow({flow: Flow.ManageBackups, step:Step.ManageBackupStep} ));`